inginious/frontend/templates/course_admin/api_tokens.html

```diff
@@ -0,0 +1,94 @@
+{# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
+{# more information about the licensing of this file. #}
+
+{% extends "layout.html" %}
+
+{% block title %} {{ course.get_name(user_manager.session_language()) }} - {{ _("Tokens") }} {% endblock %}
+
+{% block column %}
+{{ template_helper.call('course_admin_menu',course=course,current='api_tokens') | safe }}
+{% endblock %}
+
+{% block navbar %}
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="{{get_homepath()}}/course/{{course.get_id()}}">{{course.get_name(user_manager.session_language())}}</a></li>
+            <li class="breadcrumb-item"><a href="{{get_homepath()}}/admin/{{course.get_id()}}" title="{{ _('Administration') }}" data-toggle="tooltip" data-placement="bottom">
+                <i class="fa fa-user-secret"></i></a></li>
+            <li class="breadcrumb-item active"><a href="#">{<i class="fa fa-id-card"></i> { _("API tokens") }} <span class="sr-only">{{ _("(current)") }}</span></a></li>
+        </ol>
+    </nav>
+{% endblock %}
+
+
+{% block content %}
+<h2>{{_("Tokens")}}</h2>
+<div class="card-header bg-success text-white">
+    {{ _("Add token") }}
+</div>
+{% set at = course.get_accessibility(False) %}
+<div class="card-body">
+    <form class="form-horizontal" id="tokens_form" method="post">
+        <div class="row">
+            <div class="col-md-6">
+                <input type="text" placeholder="description" name="descr" value="" class="form-control">
+            </div>
+            <div class="col-md-3">
+                <div class="form-group">
+                    <div class='input-group date' id='time_picker' data-target-input="nearest">
+                        <input data-target='#time_picker' name="expiration" data-date-format="YYYY-MM-DD HH:mm:ss" value="{{at.get_std_start_date()}}" placeholder="2014-06-29 10:00" type='text' class="form-control datetimepicker-input"/>
+                        <div class="input-group-append" data-target="#time_picker" data-toggle="datetimepicker">
+                            <div class="input-group-text"><i class="fa fa-calendar"></i></div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-md-3">
+                <button class="btn btn-block btn-info" type="submit" name="add_token"><i class="fa fa-plus fa-lg"></i> {{ _("Create token") }}</button>
+            </div>
+        </div>
+    </form>
+    <script type="text/javascript">
+        $(function() {
+            $('#time_picker').datetimepicker({locale: '{{user_manager.session_language()}}', sideBySide: true, format:'YYYY-MM-DD HH:mm:ss'});
+        });
+    </script>
+     {% if msg != "" and msg != "removed" %}
+        <h4>Token string:</h4>
```

Comment:
```suggestion
        <h4>{{ _("Token string") }}:</h4>
```

Please be a little descriptive to user. Explain them to store this value because it won't be saved, etc.